### PR TITLE
CDAP-14424 Remove deprecated timetable from FileSource

### PR DESCRIPTION
**Pull request by rohitsinha54**
Issue : https://issues.cask.co/browse/CDAP-14424 

- Remove deprecated time table from FileSource

### CODE REVIEW

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileBatchSource.java [hunk 1]
```diff
@@ -59,25 +47,13 @@
   static final String LAST_TIME_READ = "last.time.read";
   static final String CUTOFF_READ_TIME = "cutoff.read.time";
```
Comment (albertshau) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileBatchSource.java]: can remove these constants too.

File: core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileBatchSource.java [hunk 2]
```diff
@@ -102,41 +78,6 @@ public Schema getSchema(FileSourceConfig config, EndpointPluginContext pluginCon
...
-      String updatedDatesToRead = GSON.toJson(attempted, ARRAYLIST_DATE_TYPE);
-      if (!updatedDatesToRead.equals(datesToRead)) {
-        table.write(LAST_TIME_READ, updatedDatesToRead);
-      }
-      properties.put(LAST_TIME_READ, datesToRead);
-    }
-
-    properties.put(CUTOFF_READ_TIME, dateFormat.format(prevHour));
     Pattern pattern = config.getFilePattern();
     properties.put(INPUT_REGEX_CONFIG, pattern == null ? ".*" : pattern.toString());
```
Comment (albertshau) [inline core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileBatchSource.java]: I think we should remove this logic as well, I believe it hooks into the time table stuff

**Review by albertshau — COMMENTED** [review]
this change also requires changes to the unit tests. Please add a link to the hydrator plugins build to verify the tests pass

**Comment by rohitsinha54** [comment]
Build: https://builds.cask.co/browse/HYP-BAD503

**Comment by albertshau** [comment]
Still need to remove the tests for the time table, but other changes look good

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**